CI: Build instructions for Coq and UniMath need to be updated

From https://github.com/UniMath/TypeTheory/actions/runs/3222458407/jobs/5271547506:

```
~/work/TypeTheory ~/work/TypeTheory/TypeTheory
Cloning into 'UniMath'...
/home/runner/work/TypeTheory/UniMath
Already on 'master'
Your branch is up to date with 'origin/master'.
commit 1966eac142152fa9d13dca25f94a74d8060eaf4c
Merge: 59ea66c6 ffdb427e
Author: Benedikt Ahrens <benedikt.ahrens@gmx.net>
Date:   Mon Oct 10 11:52:20 2022 -0400

    Merge pull request #1230 from olynch/master
    
    Created Normal Subgroups section.

--- making .coq_makefile_input
git submodule update --init sub/coq
Submodule 'sub/coq' (https://github.com/coq/coq.git) registered for path 'sub/coq'
Cloning into '/home/runner/work/TypeTheory/UniMath/sub/coq'...
Submodule path 'sub/coq': checked out '70765ee1eb2c5f1085a2c2da6a39adcac4f89b9f'
--- making sub/coq/config/coq_config.ml because of sub/coq/configure.ml
cd sub/coq && ./configure -coqide "no" -with-doc no -prefix /home/runner/work/TypeTheory/UniMath
Dune could not be found, please ensure you have a working OCaml enviroment
Makefile:79: .coq_makefile_output.conf: No such file or directory
make: *** [Makefile:247: sub/coq/config/coq_config.ml] Error 1
Error: Process completed with exit code 2.
```

--- a/.github/workflows/build-typetheory.yml
+++ b/.github/workflows/build-typetheory.yml
@@ -18,7 +18,7 @@ jobs:
   # This workflow contains a single job called "build"
   build:
     # The type of runner that the job will run on
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:

--- a/.github/workflows/build-typetheory.yml
+++ b/.github/workflows/build-typetheory.yml
@@ -30,7 +30,7 @@ jobs:
       - name: Install build dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install build-essential git ocaml ocaml-nox ocaml-native-compilers camlp5 libgtk2.0 liblablgtk-extras-ocaml-dev ocaml-findlib libnum-ocaml-dev emacs ocaml-dune
+          sudo apt-get install build-essential git ocaml ocaml-nox ocaml-native-compilers camlp5 libgtk2.0 liblablgtk-extras-ocaml-dev ocaml-findlib libnum-ocaml-dev emacs ocaml-dune libzarith-ocaml-dev
 
       # Change into UniMath directory and build UniMath
       - name: Build UniMath and TypeTheory

--- a/.github/workflows/build-typetheory.yml
+++ b/.github/workflows/build-typetheory.yml
@@ -30,7 +30,7 @@ jobs:
       - name: Install build dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install build-essential git ocaml ocaml-nox ocaml-native-compilers camlp5 libgtk2.0 libgtksourceview2.0 liblablgtk-extras-ocaml-dev ocaml-findlib libnum-ocaml-dev emacs ocaml-dune
+          sudo apt-get install build-essential git ocaml ocaml-nox ocaml-native-compilers camlp5 libgtk2.0 liblablgtk-extras-ocaml-dev ocaml-findlib libnum-ocaml-dev emacs ocaml-dune
 
       # Change into UniMath directory and build UniMath
       - name: Build UniMath and TypeTheory

--- a/.github/workflows/build-typetheory.yml
+++ b/.github/workflows/build-typetheory.yml
@@ -30,7 +30,7 @@ jobs:
       - name: Install build dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install build-essential git ocaml ocaml-nox ocaml-native-compilers camlp5 libgtk2.0 libgtksourceview2.0 liblablgtk-extras-ocaml-dev ocaml-findlib libnum-ocaml-dev emacs
+          sudo apt-get install build-essential git ocaml ocaml-nox ocaml-native-compilers camlp5 libgtk2.0 libgtksourceview2.0 liblablgtk-extras-ocaml-dev ocaml-findlib libnum-ocaml-dev emacs ocaml-dune
 
       # Change into UniMath directory and build UniMath
       - name: Build UniMath and TypeTheory

